hadoop-common-project/hadoop-common/src/test/java/org/apache/hadoop/io/TestArrayWritable.java

```diff
@@ -101,5 +108,29 @@ public void testArrayWritableStringConstructor() {
     assertArrayEquals("testArrayWritableStringConstructor toString error!!!",
       original, arrayWritable.toStrings());
   }
+
+  @Test
+  public void testArrayWritableStrings() throws Exception {
+    String[] original = {"The 1896 Cedar Keys hurricane was a powerful tropical cyclone " +
```

Comment:
don't think we should be using wikipedia text even if it's interesting. just use something from our own docs